lib/geocode.js

```diff
@@ -53,11 +52,30 @@ function _handleArcGISResponse (error, response, body, callback) {
  * geoservice.geocode({ text: "920 SW 3rd Ave, Portland, OR 97204" }, callback);
 */
 function geocode (parameters, callback) {
-  parameters.f = parameters.f || "json";
```

Comment:
> are you just going for clarity?

i didn't write that line, and the World Geocoding Service doesn't even support `html` output, but my guess is that this line was written to ensure that developers *can* ask for 'html' from their own geocoding services if they want to.

does that make sense? am i even answering your question?